src/component/Contact.js

```diff
@@ -1,49 +1,2 @@
-import Lottie from "react-lottie";
```

Comment:
This contact.js in components was showing Contact section on Home Page

Reply:
> This contact.js in components was showing Contact section on Home Page

yea i imported from pages folder not from component folder 
you want the component folder only ??